src/stores/game.js

```diff
@@ -285,6 +287,12 @@ export const useGameStore = defineStore('game', {
     },
     opponentLeft() {
       this.players = this.players.filter((player) => player.pNum === this.myPNum);
+
+      if(this.myPNum == 0) {
```

Comment:
We use === to avoid null and undefined goofery
```suggestion
      if(this.myPNum === 0) {
```